react-native-libraries.json

```
@@ -13870,5 +13870,15 @@
     ],
     "android": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/colaquecez/expo-selectable-text",
+    "examples": [
+      "https://github.com/colaquecez/expo-selectable-text/blob/main/README.md"
+    ],
+    "android": true,
+    "expoGo":true,
+    "ios": false,
```

Hello @colaquecez, thanks for adding new entry to the directory! 👍 Two suggestions below:

```suggestion
    "android": true,
    "expoGo": true,
```

Since main library link will lead to the GitHub repository anyway, we require example project of snack in `examples` array, see:
* https://github.com/react-native-community/directory?tab=readme-ov-file#examples

Also the `false` values can be skipped, it helps with data file readability:
* https://github.com/react-native-community/directory?tab=readme-ov-file#new-library-entry-template

---

Hey @Simek, thanks for the review, just updated that.